keycloak/keycloak-admin-ui
# Users: Adds create user page to users section

## Motivation


- https://issues.redhat.com/browse/APPDUX-860
- Towards https://issues.redhat.com/browse/APPDUX-861

## Brief Description

This PR adds a "Create User" page to the Users section of Keycloak. 

## Verification Steps


1. Go to Users section of Keycloak.
2. Click "Add User".
3. Verify the page matches the mock: https://marvelapp.com/prototype/7i7ja65/screen/75693207
4. Toggle help and confirm that the text content matches what is expected.
5. Click "Cancel" and make sure you are navigated back to the initial users page.
6. Click the "Add User" button again and fill in the Username.
7. Go back to the table and verify that the new user is in the table.

## Checklist:


## Additional Notes
